.github/workflows/todo.yml

```diff
@@ -0,0 +1,21 @@
+name: To Do app CI & CD
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      TOKEN: ${{ secrets.trelloToken}}
+      KEY: ${{ secrets.trelloKey}}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build test image
+      run: docker build --tag travis_todo:latest --target test .
+    - name: Run unit test
+      run: docker run travis_todo todo_app/test/test_unit.py
+    - name: Run integration test
+      run: docker run --env-file .env.test travis_todo todo_app/test/test_appTest.py
+    - name: Run end-to-end test
+      # You'll need to adjust this to also pass in the relevant environment variables
+      run: docker run --env TOKEN --env KEY travis_todo todo_app/test/test_appUiTest.py
```

Comment:
You'll also need to pass in a value for the SECRET_KEY environment variable here - you should see that Github has marked this branch as failing the tests currently and if you jump into the output it's complaining that that variable isn't set

Reply:
Acted